backend/alembic/versions/y5z6a7b8c9d0_add_unified_share_tables.py

```
@@ -0,0 +1,610 @@
+kai  # SPDX-FileCopyrightText: 2025 Weibo, Inc.
```

Ruff (0.14.14)

[warning] Found useless expression. Either assign it to a variable or remove it.

(B018)